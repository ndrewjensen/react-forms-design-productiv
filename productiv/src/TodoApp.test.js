import React from "react";
import { render } from "@testing-library/react";
import TodoApp from "./TodoApp";

let result;

describe("productiv TodoApp", function () {
  beforeEach(function () {
    result = render(<TodoApp initialTodos={[
      {
        id: 1,
        title: "Test1",
        description: "Test1 Description",
        priority: 1,
      },
      {
        id: 2,
        title: "Test2",
        description: "Test2 Description",
        priority: 2,
      },
         {
        id: 3,
        title: "Test3",
        description: "Test3 Description",
        priority: 3,
      }
    ]} 
    />);
  });


  it("renders without crashing", function () {
    render(<TodoApp  />);
  });

  it("contains expected text", function () {
    
    expect(result.getElement).toBeInTheDocument();
  });

  // it("matches snapshot", function () {
  //   const { container } = render(<TodoApp />);
  //   expect(container).toMatchSnapshot();
  // });
});