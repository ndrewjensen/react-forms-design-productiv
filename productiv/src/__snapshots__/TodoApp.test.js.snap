// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`productiv Footer matches snapshot 1`] = `
<div>
  <footer
    class="App-footer mt-5 text-muted"
  >
    <small>
      Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.
    </small>
  </footer>
</div>
`;
